addon/mixins/adapter-fetch.ts

```diff
@@ -65,7 +66,9 @@ export default Mixin.create<FetchAdapter, DS.RESTAdapter>({
 
   init() {
     this._super(...arguments);
-    deprecate('FetchAdapter is deprecated, it is no longer required for ember-data>=3.9.2', false, {
+    // TS currently fails on the `lte()` check due to https://github.com/pzuraq/ember-compatibility-helpers/issues/34
+    // @ts-ignore
+    deprecate('FetchAdapter is deprecated, it is no longer required for ember-data>=3.9.2', lte('ember-data', '3.9.1'), {
```

Comment:
Since its possible to use ember-data with the `ember-data` package, we should probably also check for `lte('@ember-data/adapter', '3.9.1')`.

Reply:
Not 100% sure, but I think the packages were only introduced in 3.11? So I guess if you use packages, this should always be false anyhow? (Not sure how `lte()` handles the addon being missing at all?)

Reply:
Ya, good point. We should test 😸

Reply:
Turns out, this will not work :grimacing: As ember-compatibility-helpers throws an error at build time if trying to check a package that is not installed. E.g.:

```
Expected "ember-fetch" to have "@ember-data/adapter" as a dependency, but it was not found.
```

So I guess this approach will not work :(

Reply:
I just used semver directly for this in a recent case. You can combine that with debug-macros if you need a build time flag but likely just a broccoli-funnel to remove the adapter if you need